README.md

```diff
@@ -56,6 +56,14 @@ Build can be done by running:
 
 The DiffKemp binary is then located in `bin/diffkemp`.
 
+For running result viewer it is necessary to have installed:
+* Node.js (14)
```

Comment:
Based on the CI, we support Node.js 14, 16, 18, and 19, right? Maybe `>= 14.x` would be better here?